rs/sns/governance/src/types.rs

```diff
@@ -1734,7 +1726,7 @@ impl Action {
         }
     }
 
-    fn proposal_criticality(&self) -> ProposalCriticality {
+    pub(crate) fn proposal_criticality(&self) -> ProposalCriticality {
```

Comment:
shouldn't ExecuteGenericNervousSystemFunction (in this function) be determined by the custom settings?

Reply:
Addressed this by removing the function.